jest.config.json

```diff
@@ -0,0 +1,19 @@
+{
+  "name": "vue-chartjs",
+  "displayName": "vue-chartjs",
+  "testEnvironment": "jsdom",
+  "setupFiles": ["<rootDir>/test/unit/setup.js"],
+  "testMatch": ["<rootDir>/test/unit/specs/(*.)spec.(js|ts)"],
+  "moduleFileExtensions": ["js", "ts", "vue"],
+  "transform": {
+    "^.+\\.vue$": "@vue/vue2-jest",
+    "^.+\\.js$": "babel-jest"
```

Comment:
Instead of babel-jest we should use swc

https://github.com/reactchartjs/react-chartjs-2/blob/8420a491f9b939f87d0d806a0d9e5a5e03043cd9/jest.config.json#L5-L16

https://github.com/reactchartjs/react-chartjs-2/blob/8420a491f9b939f87d0d806a0d9e5a5e03043cd9/package.json#L59-L61

Reply:
add swc